app/controllers/foods_controller.rb

```diff
@@ -0,0 +1,5 @@
+class FoodsController < ApplicationController
+  def index
+    @foods = FoodsService.foods_search(params[:q])
```

Comment:
Looks like you may have run out of time to create a facade, but ideally, we'd call the Facade which would then call our Service. We can think of the Service as being responsible for making the call to gather the data we need, and then our facade formatting it, creating objects out of it if needed, in order to send one object to the view with only the data we need to show.

Reply:
Currently, since you're not making objects, and extracting the data you want from the parsed JSON, you are assigning `@ foods` to the entire JSON response, which has A LOT more data than what we need to send to the view. So the facade would be responsible for taking the response from the service, iterating through the foods collection that's returned in the JSON, and creating plain old ruby objects (POROs) out of the collection, to then be stored into this instance variable.